src/renderer/lib/telemetry.js

```diff
@@ -55,41 +73,6 @@ function reset () {
   }
 }
 
-function postToServer () {
```

Comment:
Simplified this by just using `simple-get`'s friendly options. We're already requiring `simple-get` so it costs nothing.